vendor/modules.txt

```diff
@@ -1,4 +1,5 @@
 # cloud.google.com/go v0.65.0
+## explicit; go 1.11
```

Comment:
do we need explicit pin up to specific go version for vendor modules?

Reply:
earlier this go version was specified in go.mod file in each vendor directory which is moved to modules.txt after the go upgrade